packages/react-sdk/src/tree/session-storage-polyfill.tsx

```diff
@@ -0,0 +1,35 @@
+// This is a temporary work around for https://github.com/remix-run/remix/issues/3659
+
+// The code is based on this discussion https://bugs.chromium.org/p/chromium/issues/detail?id=357625
+const polyfill = `(function () {
+  try {
+    const key = "__session_storage_availability_test__";
+    sessionStorage.setItem(key, "test");
+    sessionStorage.removeItem(key);
```

Comment:
I am confused by the test and the implementation of this, is setItem forbidden or removeItem?
If both why do we need both for the test?
If both why do we not warn in all polyfilled methods?

Reply:
The test is the first line `setItem`. The second line is just a cleanup.